homeassistant/components/lightwave.py

```diff
@@ -0,0 +1,49 @@
+"""
+Support for device connected via Lightwave WiFi-link hub.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/lightwave/
+"""
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (CONF_HOST, CONF_LIGHTS, CONF_NAME,
+                                 CONF_SWITCHES)
+from homeassistant.helpers.discovery import async_load_platform
+
+REQUIREMENTS = ['lightwave==0.17']
+LIGHTWAVE_LINK = 'lightwave_link'
+DOMAIN = 'lightwave'
```

Comment:
Group this under `REQUIREMENTS` to separate out the core attributes needed by HASS from custom ones for your component (for maintainability)

Reply:
Sorry but I have no idea what you are talking about. Please point me at a component that follows a pattern that you are suggesting.